src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+    private static final int NUMBER = 100;
+
+    public static Ball getRandomBall() {
```

Comment:
Method should not be static
```suggestion
    public Ball getRandomBall() {
```